shuup/core/pricing/_priceful.py

```diff
@@ -158,6 +159,10 @@ def unit_discount_amount(self):
 
     @property
     def tax_rate(self):
+        return ensure_decimal_places(self.raw_tax_rate)
```

Comment:
How it fails? Sounds like a bug in `format_percent` then. How about fixing that instead?